.github/workflows/sdk-codegen-ci.yml

```diff
@@ -0,0 +1,179 @@
+name: sdk-codegen-ci
+
+on:
+  workflow_dispatch:
```

Comment:
You should add instructions in the README or somewhere else on how to actually trigger these.

Reply:
Added a comment at the top of the workflow file.

Reply:
Nobody is going around looking at workflow files - the docs need to be in the readme or in contributing.md